packages/charts/src/charts/line/components/lines/lines.component.tsx

```diff
@@ -141,6 +158,13 @@ const Lines = ({
     }
   }, [activeKey]);
 
+  useEffect(() => {
+    if (initialDrawFinished) {
+      lineControls.start(AnimationVariants.OffsetChange);
+      areaControls.start(AnimationVariants.OffsetChange);
```

Comment:
IMO the `areaControls` animation should be used only for enabled `areaMode`. So we could enhance `useEffect` with additional dependency on `areaMode` and add proper condition.

Reply:
Changed